Consistency 2023-02-16 (#1278)
arangodb/docs: 3.10/programs-arangodump-examples.md

  --output-directory "dump-multiple"
```

By default, _arangodump_ dumps both structural information and documents from all
non-system collections. To adjust this, there are the following command-line
arguments:

- `--dump-data <bool>`: set to `true` to include documents in the dump. Set to `false`
  to exclude documents. The default value is `true`.
- `--include-system-collections <bool>`: whether or not to include system collections
  in the dump. The default value is `false`. **Set to _true_ if you are using named
  graphs that you are interested in restoring.**

For example, to only dump structural information of all collections (including system
collections), use:

```
arangodump --dump-data false --include-system-collections true --output-directory "dump"
```

> To restrict the dump to just specific collections, there is is the `--collection` option.

Reviewer: ```suggestion
To restrict the dump to just specific collections, use the `--collection` option.
```
